modules/lang/markdown.el

```diff
@@ -5,7 +5,7 @@
 ;;; Code:
 
 (use-package markdown-mode
-  :straight nil
+  :ensure nil
```

Comment:
I believe that markdown-mode is now imported twice (agda.el and here) -- do you want them reconciled?

Reply:
This is detritus from me trying to fix the annoying behavior of `markdown-mode`, don't worry about reconciling.

I'll also note that multiple `use-package` blocks don't cause the package to get loaded twice. This confused me quite a bit when I started learning how it worked: would recommend poking around at the result of `pp-macroexpand-last-sexp` to get a feel for what is going on behind the scenes.

As an aside, I've been thinking about writing my own version of `use-package` that's a bit simpler; would be happy to discuss.